scripts/us_fed/treasury_constant_maturity_rates/extract_data.py

```diff
@@ -0,0 +1,41 @@
+USAGE = '''
```

Comment:
These will all need a license header.  Need to see if tjann@ knows which to use.

Reply:
Can copy this one: https://github.com/datacommonsorg/data/blob/master/util/__init__.py

Reply:
Fixed